app/apis/crypto_exchanges/binance_api.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+module CryptoExchanges
+  # Binance API
+  # https://github.com/binance-us/binance-official-api-docs/blob/master/rest-api.md#general-api-information
+  class BinanceApi < CryptoExchanges::BaseApi
+    BASE_URL = 'https://api.binance.us/api'
+
+    def symbol
+      "#{quote}#{base}"
+    end
+
+    def orderbook
+      faraday_client.get('v3/depth', { symbol: symbol, limit: 20 })
+    end
+
+    def orderbook_result
+      orderbook
+      response = JSON.parse(orderbook.body)
+      cee = format_orderbook_result(response)
+      { exchange: cee.exchange, max_bid: cee.min_ask, min_ask: cee.max_bid }
```

Comment:
I would just have this method return the CryptoExchanges::CryptoExchangeResult. No need to convert it to a hash here. Anything downstream can refer the this instance and call it's methods and since we'll likeely build a serializer for the class you'll want an array CryptoExchanges::CryptoExchangeResult to serialize. Also I don't see a method defined called `format_orderbook_result`. 
```suggestion
      CryptoExchanges::CryptoExchangeResult.new do |r|
        r.bids = response['bids'].map { |bid| bid[0].to_f }
        r.asks = response['asks'].map { |ask| ask[0].to_f }
        r.exchange = 'Binance.us'
      end
```

Reply:
Oops the `format_orderbook_result` was supposed to be `orderbook_result_helper` which i seperated into its own helper function because when it was in one function rubocop gave an error which they said online means there is too much happening in the function.

Reply:
will remove the assignment and the hash